spec/spec.md

```diff
@@ -479,19 +480,32 @@ Below is a non-normative example of a DID Document that includes a serviceEndpoi
 ]
 ```
 
-The issued VC MUST include a `credentialStatus` property, as defined in section 5.1 of [[ref: Status List 2021]]:
+The issued VC MAY include a `credentialStatus` property, as defined in section 5.1 of [[ref: Status List 2021]]:
```

Comment:
regardless of where status list is - ID Hub or HTTPS web resource - credentialStatus property has to be included, no?

Reply:
I would suggest that it does not. 
it is not a requirement in the VC spec, which makes sense e.g. if you have a credential which last 60s then the cost of maintaining a revocation list for it is over kill. From the verifier POV its a credential which is valid until it expires.
+ it makes it an easier lift to make an issuer.

Reply:
than I think it should be made clear that
- when statusList is used, `credentialStatus` property MUST be present, 
- when statusList is not used because credentials are short lived, `credentialStatus` property MAY not (MUST not?) be present
I would say `credentialStatus` property is CONDITIONAL

Reply:
will do but I would like to flip this, so the root of the decision is in the encapsulating spec e.g. VC's define credentialStatus, if you use that you must use status list 2021 as hosted from IDHub or https resources